### PR TITLE
added python file for basic structures for game

### LINKED ISSUE #3: Add initial structure for game

### PR DESCRIPTION
**Pull request by Lee-ArnoldIII**

### CODE REVIEW

File: go.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+'''
+All Python classes will reside in this file
```
Comment (KimjManansala) [inline go.py:15]: Hmm. Not sure about if we want all classes to reside in one file. We might have a monolith.

**Review by KimjManansala — APPROVED** [review]
Some concern on one the comment but not blocking. Lgtm

File: go.py (hunk 1 above)
Comment (Lee-ArnoldIII) [reply go.py:15]: We could move files around. I put this in as the initial storage and open to suggestions for file structure.

**Comment by Lee-ArnoldIII** [comment]
added more information for logic and set up initial structure of board. I will need to be changed to be a 9x9 board, but wanted to get the first part of logic in.

**Review by KimjManansala — APPROVED** [review]
I think this is a good start

File: go.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
+'''
+We need to design the board to be based on player skill level. Beginners = 9x9, Intermediate = 13x13, and Advanced = 19x19
```
Comment (KimjManansala) [inline go.py:2]: Nice I like this idea. We should probably also put this information in the readme

**Review by KimjManansala — APPROVED** [review]
I thinks it ready!

File: go.py (hunk 2 above)
Comment (Lee-ArnoldIII) [reply go.py:2]: I want to have a section for game rules in the readme

**Pull request merged**